tests/unit/example.spec.js

```diff
@@ -0,0 +1,12 @@
+import { shallowMount } from '@vue/test-utils'
```

Comment:
Write unit test cases for the components and routes using [https://github.com/vuejs/vue-jest](Vue-Jest)